Sources/Networking/HTTPClient/HTTPStatusCode.swift

```diff
@@ -60,6 +62,7 @@ extension HTTPStatusCode: RawRepresentable {
         case .temporaryRedirect: return 307
         case .invalidRequest: return 400
         case .unauthorized: return 401
+        case .forbidden: return 403
```

Comment:
surprised we didn't already have an endpoint returning 403 😄

Reply:
+1